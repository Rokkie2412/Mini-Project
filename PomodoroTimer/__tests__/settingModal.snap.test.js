import React from 'react';
import {create} from 'react-test-renderer';
import settingsModal from '../src/components/SetttingsModal';

jest.mock('react-native-vector-icons/Ionicons', () => 'MockedIonicons');

jest.mock('react-native-background-timer', () => 'Mocked Background Timer');

jest.mock('react-native-push-notification', () => 'Mocked Push Notification');

const tree = create(<settingsModal />);

describe('Snapshot UI test settings modal screens', () => {
  it('Snapshot', () => {
    expect(tree).toMatchSnapshot();
  });
});
