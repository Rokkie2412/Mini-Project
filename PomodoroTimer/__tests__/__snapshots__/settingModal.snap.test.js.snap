// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot UI test settings modal screens Snapshot 1`] = `<settingsModal />`;
